share/scan/apps.go

```diff
@@ -781,6 +787,16 @@ func (s *ScanApps) parseDotNetPackage(filename, fullpath string) {
 	}
 }
 
+func getDotnetModuleName(input string) string {
```

Comment:
Maybe make the capital "N" (getDot**N**etModuleName) will be more consistent with the function name "parseDotNetPackage"?
Same as other "Dotnet" naming.

Reply:
@Acmarr would you mind making the changes like @kyledong-suse suggestions?
Thanks

Reply:
Made the requested change for this function as well as the test function for consistency.